spec/bgs/services/claimant_spec.rb

```diff
@@ -31,6 +31,71 @@
     end
   end
 
+  it 'get find_assigned_flashes' do
+    VCR.use_cassette('claimant/find_assigned_flashes') do
+      response = service.claimant.find_assigned_flashes('796123232')
+
+      expect(response[:first_name]).to eq('LEWIS')
+      expect(response[:last_name]).to eq('KELLY')
+      expect(response[:middle_name]).to eq('L')
+      expect(response[:ptcpnt_id]).to eq('600043179')
+      expect(response[:return_code]).to eq('SHAR 9999')
+      expect(response[:return_message]).to eq('Records found')
+      expect(response[:vet_indicator]).to eq('Y')
+
+      first_flash = response[:flashes].first
+      expect(first_flash[:assigned_indicator]).to eq(nil)
+      expect(first_flash[:flash_name].strip).to eq('Document(s) exist in VBMS')
+      expect(first_flash[:flash_type]).to eq(nil)
+    end
+  end
+
+  it 'get find_vbms_flash' do
+    VCR.use_cassette('claimant/find_vbms_flash') do
+      response = service.claimant.find_vbms_flash('796123232')
+
+      expect(response[:assigned_indicator]).to eq('Y')
+      expect(response[:flash_name].strip).to eq('VBMS')
+      expect(response[:flash_type].strip).to eq('SV')
+    end
+  end
+
+  it 'post add_flash' do
+    VCR.use_cassette('claimant/add_flash') do
+      response = service.claimant.add_flash({ file_number: '796123232',
+                                              flash_code: 'something',
+                                              flash_station: 'here',
+                                              flash_routing_symbol: 'dontmatter' })
+      expect(response[:return]).to eq('SHAR 9999')
+    end
+  end
+
+  it 'post remove_flash' do
+    VCR.use_cassette('claimant/remove_flash') do
+      response = service.claimant.remove_flash({ file_number: '796123232',
+                                                 flash_code: '',
+                                                 flash_station: '',
+                                                 flash_routing_symbol: '' })
+      expect(response[:return]).to eq('SHAR 9999')
+    end
+  end
+
+  it 'post update_flashes' do
+    pending('Receiving Invalid data for PERSON_SPECL_STATUS_TYPE_NM error when hitting endpoint')
```

Comment:
```
     Failure/Error: raise BGS::ShareError.new(message, code)
     
     BGS::ShareError:
       ORA-20001: Invalid data for PERSON_SPECL_STATUS_TYPE_NM =
       ORA-06512: a:shrinqm:UFLASH - An unexpected error was encountered.  Please contact the System Administrator. Error is: ORA-20001: Invalid data for PERSON_SPECL_STATUS_TYPE_NM =
       ORA-06512: a:shrinqm:UFLASH
```

Reply:
I think I've made some progress on this.

Currently fluctuating between the following 2 errors:
```
Failure/Error: raise BGS::ShareError.new(message, code)
 
 BGS::ShareError:
 For input string: "600043179 "
```

which I'm not sure I have any idea what its trying to tell me... That is the "ptcpnt_id" but its not really giving me anything else to work on...

and
```
 BGS::ShareError:
 ORA-00001: unique constraint (CORPPROD.PK_PERSON_SPECL_STATUS) violate:shrinqm:UFLASH - An unexpected error was encountered. Please contact the System Administrator. Error is: ORA-00001: unique constraint (CORPPROD.PK_PERSON_SPECL_STATUS) violate:shrinqm:UFLASH
```